app/utils/whitelabel.ts

```diff
@@ -1,31 +1,75 @@
+import AskDarcelImage from '../assets/img/askdarcel.svg';
+import BackgroundImage from '../assets/img/bg.png';
+import SearchByAlgoliaImage from '../assets/img/search-by-algolia.png';
+import SFFamiliesLogo from '../assets/img/sf-families.svg';
+import SFServiceLogo from '../assets/img/sf-service.svg';
+import SFSeal from '../assets/img/sf-seal.png';
 import config from '../config';
 
-/** Whether we should display the SF Service Guide branded version of the site */
-const isSFFamiliesSite = () => window.location.host.indexOf(config.SFFAMILIES_DOMAIN) > -1;
-
-const isSFServiceGuideSite = () => !isSFFamiliesSite()
-  && window.location.host.indexOf(config.MOHCD_DOMAIN) > -1;
-
-const getSiteTitle = () => {
-  if (isSFServiceGuideSite()) {
-    return 'SF Service Guide';
-  }
-  if (isSFFamiliesSite()) {
-    return 'SF Families';
-  }
-  return 'AskDarcel';
+// Use url to get key of config object
+// @ts-ignore
+const configKey = Object.keys(config).find((key) => config[key] === window.location.host) || 'default';
+// @ts-ignore
+let configurations: any;
+
+// Configure title
+configurations.title = {
+  SFFAMILIES_DOMAIN: 'SF Families',
+  MOHCD_DOMAIN: 'SF Service Guide',
+  default: 'AskDarcel',
 };
 
-const getSiteUrl = () => {
-  if (isSFServiceGuideSite()) {
-    return 'https://sfserviceguide.org';
-  }
-  if (isSFFamiliesSite()) {
-    return 'https://sffamilies.sfserviceguide.org/';
-  }
-  return 'https://askdarcel.org';
+// Configure site url
+configurations.siteUrl = {
+  SFFAMILIES_DOMAIN: 'https://sffamilies.sfserviceguide.org/',
+  MOHCD_DOMAIN: 'https://sfserviceguide.org',
+  default: 'https://askdarcel.org',
 };
 
-export {
-  isSFServiceGuideSite, isSFFamiliesSite, getSiteTitle, getSiteUrl,
+// @ts-ignore
+const icons = require.context('../assets/img', true, /ic-.*\.(png|svg)$/i);
+const iconPathMap = {};
+
+// @ts-ignore
+icons.keys().forEach((key) => { iconPathMap[key.match(/ic-([^@]*)(?:@3x)?.(?:svg|png)/)[1]] = icons(key); });
+
+// @ts-ignore
+const icon = (name) => iconPathMap[name.toLowerCase().replace(/(\s+|\/)/g, '-')];
```

Comment:
This icon stuff doesn't look like it has anything to do with whitelabeling, since these are generic to all whitelabeled sites. Could you move this back to `assets/index.js`?

As an aside, I think we should probably get rid of this stuff at some point, since it's not really necessary in a modern frontend project, since Webpack should be able to take care of all the assets transformation stuff, so we really only need to just import the individual icons that we're using like any other Webpack image import.